holmes/plugins/toolsets/grafana/loki_api.py

```
@@ -65,10 +65,19 @@ def _make_request():
 
     try:
         response = _make_request()
+    except requests.exceptions.RequestException as e:
+        raise Exception(f"Failed to query Loki logs: {str(e)}")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)